engine/src/main/java/org/terasology/rendering/dag/RenderGraph.java

```diff
@@ -28,19 +28,14 @@ public RenderGraph() {
         nodes = Lists.newArrayList();
     }
 
-    public String add(Node node, String suggestedId) {
+    public String addNode(Node node, String suggestedId) {
         nodes.add(node);
         return null; // TODO: for instance if "blur" present make id "blur1" and return it
```

Comment:
Since value that is returned is not used anywhere and functionality of returning unique ids not there, I left it as _null_. Let me make it `return sugesstedId` in my next commit.